src/todolist.js

```diff
@@ -0,0 +1,134 @@
+export default class TodoList {
+  constructor() {
+    const savedTasks = JSON.parse(localStorage.getItem('tasks'));
+    if(savedTasks){
+      this.tasks = savedTasks;
+    }else{
+      this.tasks = [
+        {
+          description: 'task 1',
+          completed: false,
+          index: 1,
+        },
+        {
+          description: 'task 2',
+          completed: true,
+          index: 2,
+        },
+        {
+          description: 'test3',
+          completed: false,
+          index: 3,
+        },
+        {
+          description: 'test4',
+          completed: false,
+          index: 4,
+        },
+      ];
+    }
```

Comment:
- [ ] Kindly delete all hardcoded items from the tasks array as this is one of the requirements for this milestone. 
😄 💪

Reply:
done